src/Cake/Scripting/Roslyn/Stable/RoslynScriptSessionFactory.cs

```diff
@@ -10,15 +10,15 @@ namespace Cake.Scripting.Roslyn.Stable
 {
     using Core.IO;
```

Comment:
@devlead @patriksvensson @gep13 found the reason. I'ts either this or remove the *using Nuget;* since *IFileSystem* will become ambigious.

Which one do you want? Keep *using Core.IO;* inside namespace or remove *using Nuget;* and prefix all Nuget references with *global::Nuget*. (The latter is how it's done inside Scripting with Roslyn)